tests/Command/ComposerRequireCommandTest.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * This file is part of the prooph/micro-cli.
+ * (c) 2017-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2017-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ProophTest\MicroCli\Command;
+
+use PHPUnit\Framework\TestCase;
+use Prooph\MicroCli\Command\ComposerRequireCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @coversDefaultClass \Prooph\MicroCli\Command\ComposerRequireCommand
+ */
+final class ComposerRequireCommandTest extends TestCase
```

Comment:
Is that an code style rule for prooph? Maybe I have to change that on more parts.

I declare classes also final if they are DTOs or there is no need to mock they. Like test classes.